doc/source/howtos/encrypt-traffic.rst

```diff
@@ -0,0 +1,307 @@
+
+* To enable SST encryption based on OpenSSL
+  with the key (``tkey``) and certificate (``tcert``) files:
+
+  .. code-block:: text
+
+     [sst]
+     encrypt=3
+     tcert=/path/to/server.pem
+     tkey=/path/to/server.key
+
+* To enable SST encryption based on SSL files generated by MySQL,
+  use the following configuration::
+
+   [sst]
+   encrypt=4
+   ssl-ca=ca.pem
+   ssl-cert=server-cert.pem
+   ssl-key=server-key.pem
+
+  For more information, see
+  http://dev.mysql.com/doc/refman/5.7/en/creating-ssl-files-using-openssl.html
+
+  The following procedure shows how to generate the :file:`ca.pem`,
+  :file:`server-cert.pem`, and :file:`server-key.pem` files.
+
+  1. To generate the CA certificate::
+
+      openssl genrsa 2048 > ca-key.pem
+      openssl req -new -x509 -nodes -days 3600 -key ca-key.pem -out ca.pem
+
+  #. To generate the server certificate, remove passphrase, and sign it::
+
+      openssl req -newkey rsa:2048 -days 3600 -nodes -keyout server-key.pem -out server-req.pem
+      openssl rsa -in server-key.pem -out server-key.pem
+      openssl x509 -req -in server-req.pem -days 3600 -CA ca.pem -CAkey ca-key.pem -set_serial -1 -out server-cert.pem
+
+  #. (Optional) To generate the client certificate,
+     remove passphrase, and sign it::
+
+      openssl req -newkey rsa:2048 -days 3600 -nodes -keyout client-key.pem -out client-req.pem
+      openssl rsa -in client-key.pem -out client-key.pem
+      openssl x509 -req -in client-req.pem -days 3600 -CA ca.pem -CAkey ca-key.pem -set_serial 01 -out client-cert.pem
+
+  There are two ways to deploy SSL files across the cluster:
+
+  * Use the same files for all machines in the configuration.
+    Generate the files on one machine (or use the MySQL-generated files),
+    and copy the files to each node in the cluster.
+    Follow steps (1) and (2) in the previous procedure
+    to manually generate the files.
+
+  * Generate one CA file with separate server certificates
+    signed by that one CA file.
+    Do step (1) to generate the CA file
+    and then do step (2) for each server using the same CA file.
+    So each server will have different :file:`server-cert.pem` files,
+    but they will all share the same :file:`ca.pem` file.
+
+  .. note:: Whatever method you use to generate the certificate and key files,
+     the ``Common Name`` value used for the server and client certificates/keys
+     must each differ from that value used for the CA certificate.
+     Otherwise, the certificate and key files will not work for servers
+     compiled using OpenSSL.
+
+     The easiest way to do this is to give the CA file a common name (CN)
+     as follows::
+
+      openssl req -new -x509 -nodes -days 3600 -key ca-key.pem -out ca.pem -subj “/CN=my_cluster_name”
+
+.. note:: Percona XtraBackup supports keyring transfer
+   in version 2.4.4 and later.
+
+.. note:: SSL clients require DH parameters to be at least 1024 bits.
```

Comment:
We need to be more clear here.
Describing step.

@azhebel 
Also, it is difficult to imagine how this appear in html format but it should be much organized so that
structure of page help end-user capture the meaning.